pinot-core/src/main/java/org/apache/pinot/core/data/order/OrderByUtils.java

```diff
@@ -109,12 +110,33 @@ private OrderByUtils() {
     return globalComparator;
   }
 
+  public static List<Integer> getAggregationIndexes(List<SelectionSort> orderBy, List<AggregationInfo> aggregationInfos) {
```

Comment:
It's not entirely similar. We can end up with 0 length list or upto numAggregations length list. To return int[], i'll still have to begin with list, and then make it an array, or scan twice to avoid that.

Reply:
That is okay. The benefit of using int[] is the data access will be much faster. After calculating the indices, we need to access this array on a per value basis

Reply:
done